source/security/authentication.txt

```diff
@@ -0,0 +1,390 @@
+.. _php-auth:
+
+=========================
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: authorize, secure, connect, code example
+
+Overview
+--------
+
+This guide describes the mechanisms you can use in the {+driver-short+} to authenticate
+users.
+
+.. important:: Percent-Encoding
+
+   You must :wikipedia:`percent-encode <Percent-encoding>` a username and password before
+   you include them in a MongoDB URI. You can use the ``rawurlencode()`` method to encode 
+   these values according to the URI syntax specified in RFC 3986. Don't percent-encode the
```

Comment:
Did you mean to use the link you defined for "RFC 3986" below?

Reply:
We try to avoid inline links that aren't introduced, and since there's already one to Wikipedia I just linked in the list below

Reply:
Oh, I missed that those were part of a list and assumed they were internal references that didn't get used. Disregard my earlier comment.